integration/src/test/java/org/mule/shutdown/ValidShutdownTimeoutOneWayTestCase.java

```diff
@@ -40,7 +40,7 @@
   private TestQueueManager queueManager;
 
   @Rule
-  public SystemProperty contextShutdownTimeout = new SystemProperty("contextShutdownTimeout", "" + RECEIVE_TIMEOUT);
+  public SystemProperty contextShutdownTimeout = new SystemProperty("contextShutdownTimeout", "1" + RECEIVE_TIMEOUT);
```

Comment:
?

Reply:
```suggestion
  public SystemProperty contextShutdownTimeout = new SystemProperty("contextShutdownTimeout", "" + RECEIVE_TIMEOUT);
```
Idk where this came from...